META

```diff
@@ -6,5 +6,5 @@ Release:       1
 Release-Tags:  relext
 License:       CDDL
 Author:        OpenZFS
-Linux-Maximum: 6.2
+Linux-Maximum: 6.3
```

Comment:
Shouldn’t this be 6.4 like the title?

Reply:
@RaitoBezarius if you fix up the version here and rebase it against the latest zfs-2.1.13-staging branch we can get this applied so it's queued up for the next point release.

Reply:
@behlendorf Done!